install/teddycloud-install.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 community-scripts ORG
+# Author: Dominik Siebel (dsiebel)
+# License: MIT
+# https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+
+source /dev/stdin <<< "$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get install --no-install-recommends --yes \
+  curl \
+  sudo \
+  mc \
+  libubsan1 \
+  ffmpeg \
+  curl \
+  ca-certificates
+msg_ok "Installed Dependencies"
+
+msg_info "Installing TeddyCloud"
+RELEASE="$(curl -s https://api.github.com/repos/toniebox-reverse-engineering/teddycloud/releases/latest | grep "tag_name" | awk '{print substr($2, 2, length($2)-3) }')"
+VERSION="${RELEASE#tc_v}"
```

Comment:
you can do  for example RELEASE:1 to crop the first. Can you check it? Then you dont need VERSION here

Reply:
Yes, I could, but I'd have to do that in multiple places and keep it consistent.
This way, `VERSION` always contains the "clean" tag, while `RELEASE` contains the whole tag.
Both need to be referenced in the download URL for example.

Happy to change it, though, if you insist.